apps/condo/domains/notification/tasks/sendMessageBatch.helpers.js

```diff
@@ -0,0 +1,101 @@
+const { get, isString } = require('lodash')
+
+const conf = require('@condo/config')
+
+const { getLogger } = require('@condo/keystone/logging')
+
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+
+const {
+    SMS_TRANSPORT, EMAIL_TRANSPORT, PUSH_TRANSPORT,
+    CUSTOM_CONTENT_MESSAGE_TYPE,
+} = require('../constants/constants')
+
+const EMAIL_FROM = 'noreply@doma.ai'
+const DATE_FORMAT = 'YYYY-MM-DD'
+const IS_EMAIL_REGEXP = /^\S+@\S+\.\S+$/
+const IS_PHONE_REGEXP = /^\+79\d{9}$/
+const IS_UUID_REGEXP = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i
```

Comment:
maybe, move all regexps to constants?

IS_PHONE_REGEXP = +7 // Only Russia ?

Reply:
by now we support only Russian mobile phones for sms